force-app/main/default/classes/TaskTriggerHelperTest.cls

```diff
@@ -0,0 +1,32 @@
+@isTest
+public class TaskTriggerHelperTest {
+  public static List<Task> createTasks(
+    List<Account> accounts,
+    String taskType,
+    Integer numOfTasks
+  ) {
+    List<Task> tasks = new List<Task>();
+    for (Account acc : accounts) {
+      for (Integer i = 0; i < numOfTasks; i++) {
+        Task newTask = new Task(Type = taskType, WhatId = acc.id);
+        tasks.add(newTask);
+      }
+    }
+    insert tasks;
+    return tasks;
+  }
+
+  public static List<Task> createTasksOnSingleAccount(
```

Comment:
createTasksOnSingleAccount vs createTasks

one method could at least call the other. 

I dont think two methods are warranted, really :) a list of accounts with 1 account in it does the same as createTasksOnSingleAccount

Reply:
And for one single method, I don't create yet another class.

Reply:
Oh ok. You spoke about creating a helper so I was thinking about a helper class for creating this method. I see now that you explicitly wrote 'method'. My bad.